egs/mgb2/asr/local/mgb_data_prep.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2020 Kanari AI 
+# (Amir Hussein)
+
+
+if [ $# -ne 3 ]; then
+  echo "Usage: $0 <DB-dir> <process-xml>"
+  exit 1;
+fi
+mer=80
```

Comment:
How about define the $mer in main script run.sh and pass to mgb_data_prep.sh as variable? It should be more flexible once someone want to change it.

Reply:
Done